tools.go

```diff
@@ -1,8 +0,0 @@
-// +build tools
-
-package main
-
-import (
-	_ "github.com/client9/misspell/cmd/misspell"
-	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
-)
```

Comment:
This underscore imports are how you vendor dev tools. Since we're not vendoring anymore, these should be deleted.